lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -102,6 +104,49 @@ sub register {
   );
 }
 
+sub _check_for_wildcards {
+    my ( $self, $files ) = @_;
+
+
+    #Keep track of files that we have seen already
+    my %seenFiles;
```

Comment:
Nice! I think I would probably forget about this... However, should we warn() if we see the same file twice..?